README.md

```diff
@@ -1,2 +1,27 @@
-# srmgpa
-Calculates SRM UNIVERSITY Semester Grade Point Average. Feel free to raise a Pull Request if the formula has changed or if you can make it better.
+# SRM GPA Calculator
+
+A simple and easy to use web application for calculating your SGPA based on credits and grades in courses.
+
+
+## Introduction
+
+The GPA Calculator is a user-friendly tool that helps you quickly calculate your Semester Grade Point Average (SGPA) following the Grading rules of SRM University. It takes into account the Credits and Grades for each course, and provides you with an accurate SGPA.
+
+## Features
+
+- Easy input of Credits and Grades for each subject.
+- Calculates GPA based on the provided data.
+- Allows for adding more courses dynamically.
+- Reset option to start over.
+
+## Demo
+
+You can try out this GPA Calculator by visiting this [link](https://optimusam.github.io/srmgpa/). Simply enter your course details and hit the "Submit" button to see your SGPA.
+
+## Contributing
+Contributions are always welcome! Feel free to contribute to this project by creating a Pull Request.
+
+## License
+
+[![MIT License](https://img.shields.io/badge/License-MIT-green.svg)](https://choosealicense.com/licenses/mit/)
```

Comment:
Please remove the license part as I've already added GNU Affero General Public License v3.0. It's more open-source friendly.

Reply:
Done!